tools/llvm_clean.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# A script for cleaning up the environment after upgrading LLVM.
+#
+# Usage:
+# ./llvm_clean.sh [DIFFKEMP_ROOT]
+#
+# When DIFFKEMP_ROOT is not provided, the script assumes that
+# the current working directory is the DiffKemp root directory.
+
+if [[ -z $1 ]]
+    then
+        DIFFKEMP_ROOT=$PWD
+    else
+        DIFFKEMP_ROOT=$1
+fi
+
+if [[ ! -f ${DIFFKEMP_ROOT}/diffkemp/simpll/_simpll.abi3.so ]]
```

Comment:
There might be an issue with the tool not printing anything when the path is wrong, e.g. because of a typo, since there will be no indication of an error in that case.

It might make sense to check for the existence of e.g. `${DIFFKEMP_ROOT}/diffkemp/simpll` first, and exit with an error if it doesn't exist.

Reply:
Thanks, that's a good idea! It should be fixed now.